### DISCUSSION TITLE
Serial Console sending Data Every 10-Seconds

### DISCUSSION THREAD

**Discussion by ChetSzatkowski-MCD** [cncjs/cncjs#835] [General]
CNCjs (standalone Windows x64) is working as expected and is connected to a GRBL 1.1 CNC controller.

However, when the GRBL controller is "Active" I am receiving a command and "ok" response every 10-seconds.

I've verified that it is not the GRBL "controller" sending tese commands/responses by connecting serially directly to the controller using PuTTY.

PuTTY is silent (as expected).

Here is the sample output which appears on the "console" every 10-seconds:

GC:G0 G54 G17 G21 G90 G94 M5 M M9 T0 F0 S0]
ok

**Comment by MitchBradley** [comment]
CNCjs is sending periodic $G commands to ask for the GCode state.  The response is supposed to start with [.  Apparently that is getting lost somehow.

If you send $G with PuTTY, you should see a similar response.  It really should be
[GC:G0 G54 G17 G21 G90 G94 M5 M M9 T0 F0 S0] - with an initial [
